core/components/atoms/tooltip/tooltip.md

```diff
@@ -9,23 +9,38 @@ description: Use tooltips for giving extra context AND to make visual cues acces
 <Tooltip defaults={{content: "Here is some text"}} {props}>Hover me</Tooltip>
 ```
 
-## Tooltip Positions
+## Examples
+
+### Tooltip Positions
 
 Tooltips can be attached to any of the four sides of an element using the `position` prop.
 
 ```js
 <Stack>
   <Tooltip position="top" content="Top">
-    <Icon name="copy" />
+    <Icon size={16} name="help" />
```

Comment:
at `size` prop 

Why is this needed in the example? 🤔

Reply:
Just to make the icon smaller and how a help tooltip would look like

Reply:
No harm using a default.